vue-app/src/components/CartItems.vue

```diff
@@ -25,6 +30,7 @@
           {{item.amount}} {{tokenSymbol}}
         </div>
       </div>
+      <!-- TODO: if in tallying phase and the user contributed to matching pool as part of reallocation we should list it here -->
```

Comment:
@pettinarip is this a Ryan comment? Not really sure what this means.

Reply:
Yes, it's from Ryan. I don't know either. Maybe it is related to the "old" design where we were going to have a Matching pool item?